backend_new/src/controllers/listing.controller.ts

```diff
@@ -65,4 +67,18 @@ export class ListingController {
       queryParams.view,
     );
   }
+
+  @Get(`byMultiselectQuestion/:multiselectQuestionId`)
```

Comment:
@YazeedLoonat , how will this endpoint get used?

Reply:
sure, on the preference settings page in partners we have a delete action, we use this to display the listings that are tied to that  "preference" which in reality is a multiselect question